descope/management/user_pwd.py

```diff
@@ -0,0 +1,130 @@
+from typing import Optional, Union
+
+
+class UserPasswordBcrypt:
+    def __init__(
+        self,
+        hash: str,
+    ):
+        """
+        The bcrypt hash in plaintext format, for example "$2a$..."
+        """
+        self.hash = hash
+
+    def json(self):
```

Comment:
more pythonic way is to define this function like this:
`def to_dict(self) -> dict:`